README.md

```diff
@@ -101,6 +101,38 @@ open-attestation encrypt ./examples/wrapped-certs/example.0.json ./tmp/encrypted
 ⚠  warning   Here is the key to decrypt the document: don't lose it: 9bac5be27bac31d852fc1e48eb9d5249ec6ad7978da23377b5879f7a24994cb2
 ```
 
+## Deploying Token Registry
```

Comment:
not for myself, what happen if nothing is provided (no key-file, no key, no env)

Reply:
I wont want to write the whole exceptions flow here. I think the document should tell them how to do it, not what happens if you try something shitty lol

Reply:
added test on missing keys instead

Reply:
yes the not is really for myself to test this out :+1: